backend/execution_cache/go.mod

```diff
@@ -0,0 +1,9 @@
+module src
```

Comment:
please follow other backend code and put go.mod/sum under the root 
https://github.com/kubeflow/pipelines/blob/master/go.mod

Reply:
Would it be beneficial to go with microservice approach where every service has its own code, image and packages?

Reply:
what i was proposing is just to make the code location consistent. the binary and container should still be on its own.